drone/secret/secret_add.go

```diff
@@ -52,29 +75,67 @@ func secretCreate(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	secret, err := makeSecret(c)
+	if err != nil {
+		return err
+	}
+	if len(secret.Events) == 0 {
+		secret.Events = defaultSecretEvents
+	}
+	_, err = client.SecretCreate(owner, name, secret)
+	if err == nil {
+		return printSecret(secret, tmplSecretList)
```

Comment:
this is true although going forward, it might be nice to print output when resources are created or updated to give the user visual confirmation ... similar to other tools like git for example:

```term
$ git commit -m "update README"
[master 10f26b6] update store
18 files changed, 514 insertions(+), 238 deletions(-)
```